content/ch-algorithms/deutsch-jozsa.ipynb

```diff
@@ -199,44 +199,44 @@
     "    <li> The first register of two qubits is initialized to $|00\\rangle$ and the second register qubit to $|1\\rangle$ \n",
     "    \n",
     "\n",
-    "$$\\lvert \\psi_0 \\rangle = \\lvert 0 0 \\rangle_1 \\lvert 1 \\rangle_2 $$\n",
+    "$$\\lvert \\psi_0 \\rangle = \\lvert 0 0 \\rangle \\otimes \\lvert 1 \\rangle $$\n",
     "\n",
     " \n",
     "   </li>\n",
     "    \n",
     "   <li> Apply Hadamard on all qubits\n",
     "    \n",
     "\n",
-    "$$\\lvert \\psi_1 \\rangle = \\frac{1}{2} \\left( \\lvert 0 0 \\rangle_1 + \\lvert 0 1 \\rangle_1 + \\lvert 1 0 \\rangle_1 + \\lvert 1 1 \\rangle_1 \\right) \\frac{1}{\\sqrt{2}} \\left( \\lvert 0 \\rangle_2 - \\lvert 1 \\rangle_2 \\right)  $$\n",
+    "$$\\lvert \\psi_1 \\rangle = \\frac{1}{2} \\left( \\lvert 0 0 \\rangle + \\lvert 0 1 \\rangle + \\lvert 1 0 \\rangle + \\lvert 1 1 \\rangle \\right) \\otimes \\frac{1}{\\sqrt{2}} \\left( \\lvert 0 \\rangle - \\lvert 1 \\rangle \\right)  $$\n",
     "\n",
     " \n",
     "   </li>\n",
     "    \n",
     "   <li> The oracle function can be implemented as $\\text{Q}_f = CX_{1a}CX_{2a}$, \n",
```

Comment:
Can we explain the subscripts here please? I think `1a` means control is qubit 1, target is auxillary qubit (called output qubit in the rest of the article)? But this wasn't immediately obvious to me.

Reply:
Line 215? Yeah, I missed that sorry. I'll try and make it clearer. It's pretty vague right now what the oracle does and how it maps states.

Reply:
But first:
The example says `Let's go through a specific example for a two bit balanced function, where we apply X-gates before both CNOTs`. However, the X gates aren't explicitly applied anywhere. I know that applying X gates on both qubits when they're already in the |++> state is equivalent to doing nothing but maybe we should reword that?

Reply:
hmm yes that is strange. Without working through the equation properly, I can't see it has any effect on the maths either, maybe we should just remove the line?

Reply:
Yeah I think we can safely remove it. Let me just take a look at it in its entirety once and ensure that everything makes sense.

Reply:
@frankharkins I've changed it a little, let me know your thoughts!

Reply:
Looks good thank you!